src/helpers.ts

```diff
@@ -29,7 +29,13 @@ function chordTransposeDistance(capo: number, transposeKey: string | null, songK
   return transpose;
 }
 
-export function renderChord(chordString: string, line: Line, song: Song, renderKey: Key | null = null): string {
+export function renderChord(
+  chordString: string,
```

Comment:
@martijnversluis Would you like me to put all these params into an object, or something else?

Reply:
So far no better solution came to mind, so indeed an object would make the most sense.

I do think I'd prefer the `use` prefix for all parameters, so `useUnicodeModifiers` for the property in `Configuration` and for the function parameter for `renderChord`, and `useUnicodeModifier` for `Chord.toString()`.

Reply:
> Wow @jbree. Real nice PR, thanks for your great work! I'll have a thought about the `renderChord` function, because it getting too many params is a recurring issue. Will get back to you 👍

Thanks for the feedback! Will hopefully have time to make requested changes within a day or two.